cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -46,7 +46,8 @@ set -o nounset
 BUNDLED_VERSIONS="3.0.17, 3.1.12, 3.2.24, 3.3.17, 3.4.7"
 
 # shellcheck disable=SC2039
-ETCD_NAME="${ETCD_NAME:-etcd-${ETCD_HOSTNAME:-$(hostname -s)}}"
+# NOTE: Keep ETCD_NAME in sync with host_name in https://github.com/kubernetes/kubernetes/blob/2e7996e3e2712684bc73f0dec0200d64eec7fe40/cluster/gce/gci/configure-helper.sh#L1563
```

Comment:
Now let's remove this comment, since it is no longer valid really.

Reply:
I think the bulk part is still "in sync", although the definition text is not the same, the resulting etcd name are the same. And the text is different only because hostname binary is not available anymore. We still want to make sure if etcd name is changed in configure-helper.sh, it should be changed here too.

Reply:
So let's make this comment very explicit - currently someone will come and unify it.